docs/ADR/001-porting-to-postgresql.md

```diff
@@ -0,0 +1,47 @@
+# ADR 001 - How we migrate Router/Router-API from Mongo to Postgres
+
+## Context
+
+We need to migrate router (and [router-api](https://github.com/alphagov/router-api)) off of MongoDB as its datastore. The majority of GOV.UK platform is on PosgreSQL and there is no longer a strong enough case for being on MongoDB for router data.
+
+Even the [content-store is migrating from MongoDB to PostgreSQL](https://github.com/alphagov/govuk-rfcs/pull/158).
+
+This ADR is to outline our decisions and rationale on the steps we take for that migration:
+
+### The outline
```

Comment:
I think this is the "Decision".

Reply:
This should be called "Decision". And be a Level 2 header.

Reply:
Sorry misunderstood you before - done